dpgen/data/surf.py

```diff
@@ -477,13 +477,13 @@ def pert_scaled(jdata) :
           raise RuntimeError("the length of vacuum_resol must equal 1 or 2")
           
     else:         
-       vacuum_num = jdata['vacuum_numb']
-       head_ratio = jdata['head_ratio']
-       mid_point = jdata['mid_point']
+       vacuum_num = jdata['vacuum_numb'] # the total number of vacuum layers
+       head_ratio = jdata['head_ratio'] # the nearby region with denser intervals (head region)
```

Comment:
\# deciding the mid_point by `vacum_max` * `head_ratio`, which poiny separates the nearby region with denser intervals (head region) and the far-away region with sparser intervals (tail region).